services/likes_service.py

```
@@ -21,16 +23,17 @@ def toggle_like_service(user_id: str, like_in: LikeCreate):
             "created_at": datetime.now(timezone.utc).isoformat()
         }
         likes.append(new_like_data)
+        like_id = new_like_data["like_id"]
 
-    print(likes)
     save_likes(likes)
 
-    current_total = len([l for l in likes if l["target_id"] == like_in.target_id])
+    current_total = len([l for l in likes if l["target_id"] == like_in.target_id
```

Ruff (0.14.13)

Ambiguous variable name: `l`

(E741)